lib/refinement.rb

```diff
@@ -1,6 +1,27 @@
-require 'refinement/version'
+require 'xcodeproj'
 
+# Generates a list of Xcode targets to build & test as a result of a git diff.
 module Refinement
   class Error < StandardError; end
-  # Your code goes here...
+
+  # @visibility private
+  # @param enum [Enumerable]
+  # Enumerates through `enum`, and applied the given block to each element.
+  # If the result of calling the block is truthy, the first such result is returned.
+  # If no such result is found, `nil` is returned.
+  def self.map_find(enum)
```

Comment:
nit: can you rename it to `map_find_first`? That's what it seems to be doing.

Reply:
`Enumerable#find` returns the first occurrence passing the block, so I was just matching that naming